script.js

```diff
@@ -2,3 +2,83 @@ var main = function (input) {
   var myOutputValue = 'hello world';
   return myOutputValue;
 };
+// set global var//
+var win = 0;
+var lose = 0;
+var draw = 0;
+var troll = 0;
+
+// hack function//
+var Hack = function (input) {
+  if (input == 'Reverse') {
+    troll += 1;
+    return 'troll activated';
+  }
+};
+
+// unhack function//
+var Hack = function (input) {
+  if (input == 'Unhack') {
+    troll -= 1;
+    return 'troll deactivated';
+  }
+};
+
+// normal function//
+var normal = function (input) {
```

Comment:
Function names should start with verb for clarity that the variable is a function. https://bootcamp.rocketacademy.co/course-logistics/naming-casing-and-commenting-conventions